bus/bus.go

```diff
@@ -1711,8 +1712,16 @@ func (b *bus) gougingParams(ctx context.Context) (api.GougingParams, error) {
 	}, nil
 }
 
-func (b *bus) handleGETAlerts(c jape.Context) {
-	c.Encode(b.alertMgr.Active())
+func (b *bus) handleGETAlerts(jc jape.Context) {
+	var offset, limit uint64
+	if jc.DecodeForm("offset", &offset) != nil {
+		return
+	} else if jc.DecodeForm("limit", &limit) != nil {
+		return
+	} else if limit == 0 {
+		limit = math.MaxUint64
```

Comment:
This behaviour is inconsistent with the rest of our API. Limit and offset are usually `int`'s and limit defaults to -1.